kraken/lib/layers.py

```diff
@@ -2,7 +2,7 @@
 Layers for VGSL models
 """
 import logging
-from typing import Iterable, List, Optional, Tuple
+from typing import Iterable, Optional
```

Comment:
These changes make it impossible to run with Py38. I know it's EOL, but is that really necessary at this point?